variables.tf

```diff
@@ -79,6 +79,12 @@ variable "apiary_log_prefix" {
   default     = ""
 }
 
+variable "enable_metrics" {
```

Comment:
maybe we rename this variable as `enable_metastore_metrics` or `enable_hive_metastore_metrics` because `enable_metrics` is generic and lead to confusion in future if we add support of metrics for any other component.

Reply:
Agreed. `enable_hive_metastore_metrics ` gets my vote.